### DISCUSSION TITLE
Sympy.physics.mechanics

### DISCUSSION THREAD

**Discussion by Peter230655** [sympy/sympy#21420] [Q&A]
I am trying to learn the GREAT sympy.physics.mechanics module, or whatever you call it.
Doing so I came across something like this:

a, b, c = dynamicsymbols('a, b, c')
N = ReferenceFrame('N')
P0 = Point('P0')
A = N.orientnew('A', 'Body', (a, b, c), '123')
Z = N.variable_map(A)

Z is a dictionary holding the coordinates of A relative to N.
How can I ‚lambdify‘ this dictionary so I can do numeric calculation?

Any hint is greatly appreciated!

**Comment by moorepants** [comment]
You probably want to lambdify the result of `A.dcm(N)` which gives you a rotation matrix.

**Comment by Peter230655** [comment]
Dear Jason,

Thanks!!
I will try this.

Peter

On Mon 3. May 2021 at 15:02 Jason K. Moore ***@***.***> wrote:

> You probably want to lambdify the result of A.dcm(N) which gives you a
> rotation matrix.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sympy/sympy/discussions/21420#discussioncomment-687831>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT5MQURI6V7LM3HZU5RECU3TL2NHDANCNFSM437OSH5A>
> .
>
-- 
Best regards,

Peter Stahlecker
